File: frontend/stockearte/src/app/shared/types/Producto.ts
export interface ProductoArray {
  productos: Producto[]
}

export interface Producto {
  tiendaIds: any[]
  nombre: string
  codigo: string
  color: string
  talle: string
  habilitado: boolean
  cantidad: number
  id: string | number
  foto: string
  idUserAdmin: string
}

File: backend-servidor/soapsys/src/main/java/com/unla/soapsys/model/Producto.java
package com.unla.soapsys.model;

import jakarta.persistence.*;

@Entity
@Table(name = "productos")
public class Producto {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String nombre;
    private String codigo;
    private String talle;
    private String foto;
    private String color;
    private int cantidad;

    // Getters y setters...
}
